src/components/NewSnippet.jsx

```diff
@@ -1,60 +1,49 @@
 import React from 'react';
-import { Scrollbars } from 'react-custom-scrollbars';
 import { Controlled as CodeMirror } from 'react-codemirror2';
 
 import 'codemirror/lib/codemirror.css';
 
 import Title from './common/Title';
 import Input from './common/Input';
+import Syntaxes from './Syntaxes';
 
 import '../styles/NewSnippet.styl';
 
-const NewSnippet = () => (
-  [
-    <Title title="New snippet" key="title" />,
-    <div className="new-snippet" key="new-snippet">
-      <div className="new-snippet-code-wrapper">
-        <div className="new-snippet-code-header">
-          <Input placeholder="Title" />
-          <Input placeholder="Tags (separate tags by comma)" />
-        </div>
-        <div className="new-snippet-code">
-          <CodeMirror
-            value="console.log('All hail XSnippet')"
-            options={{ lineNumbers: false }}
-          />
-          <div className="new-snippet-code-bottom-bar">
-            <button>POST</button>
+class NewSnippet extends React.Component {
+  constructor() {
+    super();
+    this.state = { code: '' };
```

Comment:
I think `code` is a little bit ambiguous. It's not necessary code.. Let's call it `snippet` instead or `content`.